packages/grid/_modules_/grid/models/params/gridCellParams.ts

```diff
@@ -82,9 +82,54 @@ export interface GridRenderEditCellParams extends GridEditCellProps {
 }
 
 /**
- * Alias of GridRenderCellParams.
+ * Parameters passed when calling `colDef.valueGetter` in the rendering sequence.
  */
-export type GridValueGetterParams = Omit<GridRenderCellParams, 'formattedValue' | 'isEditable'>;
+export interface GridValueGetterFullParams<V = any, R = any>
+  extends Omit<GridCellParams<V, R>, 'formattedValue' | 'isEditable' | 'getValue'> {
+  /**
+   * Get the cell value of a row and field.
+   * @param {GridRowId} id The row id.
+   * @param {string} field The field.
+   * @returns {GridCellValue} The cell value.
+   * @deprecated Use `params.row` to directly access the fields you want instead.
+   */
+  getValue: (id: GridRowId, field: string) => GridCellValue;
+  /**
+   * GridApi that let you manipulate the grid.
+   */
+  api: GridApi;
+}
+
+/**
+ * Parameters passed when calling `colDef.valueGetter` in the row grouping sequence.
+ */
+export interface GridValueGetterSimpleParams<R = any> {
```

Comment:
I don't think it's worth to create another interface to remove `getValue`. In v6 we'll need to remove it. Instead, we could only flag it as deprecated and be cautious to never call it + display warning if someone calls it.

Reply:
This interface has other differences than `getValue`
Once again it's for the grouping process, so it does not have `api` either for instance.

Reply:
> If you want to fully deprecate getValue, I can move it's deprecation from GridValueGetterFullParams to GridCellParams. That was not the goal of this PR but I'm not agaist the idea.

Yes, let's deprecate `getValue` entirely. The sooner we do, less work we have for v6.

> But for GridValueGetterSimpleParams I think we do need another interface.

For now, it's necessary because the `api` is a required property. However, in v6 this interface can be removed.